docs/java-rest/high-level/document/exists.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[java-rest-high-document-exists]]
+=== Exists API
+
+[[java-rest-high-document-exists-request]]
+==== Exists Request
+
+The `exists()` API is `true` if a document exists, and `false` otherwise.
```

Comment:
nit: is -> returns

Reply:
Hm... In most docs there is no explanation what the API is for ( just how to use it ) 
Should this be moved into the above section?

Reply:
Yeah, I should move it to the section above.

I can use `returns` instead of `is`, sure. I wanted to use `is` because async calls don't really return, but `is` is much less clear. `returns` is almost certainly better.